src/components/themes/purple-haze/theme.css

```diff
@@ -0,0 +1,476 @@
+.skinHeader,
+html {
+    color: #ddd;
+    color: rgba(255, 255, 255, .8)
+}
+
+.wizardStartForm,
+.ui-corner-all,
+.ui-shadow {
+    background-color: #303030
+}
+
+.emby-collapsible-button {
+    border-color: #383838;
+    border-color: rgba(255, 255, 255, .135)
+}
+
+.skinHeader-withBackground {
+    background: #303030;
+    background: -webkit-gradient(linear, left top, right top, from(#000420), color-stop(#06256f), color-stop(#5a1259), color-stop(#06256f), to(#011145));
+    background: -webkit-linear-gradient(left, #000420, #06256f, #5a1259, #2b052b, #011145);
+    background: -o-linear-gradient(left, #000420, #06256f, #5a1259, #2b052b, #011145);
+    background: linear-gradient(to right, #000420, #06256f, #5a1259, #2b052b, #011145)
+}
+
+.skinHeader.semiTransparent {
+    -webkit-backdrop-filter: none !important;
+    backdrop-filter: none !important;
+    background-color: rgba(0, 0, 0, .3);
+    background: -webkit-gradient(linear, left top, left bottom, from(rgba(0, 0, 0, .6)), to(rgba(0, 0, 0, 0)));
+    background: -webkit-linear-gradient(rgba(0, 0, 0, .6), rgba(0, 0, 0, 0));
+    background: -o-linear-gradient(rgba(0, 0, 0, .6), rgba(0, 0, 0, 0));
+    background: linear-gradient(rgba(0, 0, 0, .6), rgba(0, 0, 0, 0))
+}
+
+.pageTitleWithDefaultLogo {
+    background-image: url(../logowhite.png)
+}
+
+.dialog,
+html {
+    background-color: #3f2d7a
+}
+
+.backgroundContainer {
+    background: url(bg.jpg) center top no-repeat #030322;
+    -webkit-background-size: cover;
+    background-size: cover
+}
+
+.backgroundContainer.withBackdrop {
+    opacity: .93
+}
+
+@media (orientation:portrait) {
+    .backgroundContainer {
+        background-position: 30% top
+    }
+}
+
+.paper-icon-button-light:focus {
+    color: #00a4dc;
+    background-color: rgba(0,164,220, .2)
+}
+
+.fab,
+.raised {
+    background: rgba(0, 0, 0, .5);
+    color: rgba(255, 255, 255, .87)
+}
+
+.fab:focus,
+.raised:focus {
+    background: rgba(0, 0, 0, .7)
+}
+
+.button-submit {
+    background: #00a4dc;
+    color: #fff
+}
+
+.button-submit:focus {
+    background: #0cb0e8;
+    color: #fff
+}
+
+.checkboxLabel {
+    color: inherit
+}
+
+.checkboxListLabel,
+.inputLabel,
+.inputLabelUnfocused,
+.paperListLabel,
+.textareaLabelUnfocused {
+    color: #bbb;
+    color: rgba(255, 255, 255, .7)
```

Comment:
I guess having 2 `color` is a typo?

Reply:
Not all browsers support RGBa, so if the design permits, you should declare a "fallback" color. This color will be most likely be solid (fully opaque). Not declaring a fallback means no color will be applied in browsers that don't support it. This fallback does fail in some really old browsers.